traits.rs

```diff
@@ -135,3 +135,38 @@ pub trait PSP22Metadata {
     #[ink(message)]
     fn token_decimals(&self) -> u8;
 }
+
+#[ink::trait_definition]
+pub trait PSP22Burnable {
+    /// Burns `value` tokens from the senders account.
+    ///
+    /// # Events
+    ///
+    /// On success a `Transfer` event is emitted with `None` recipient.
+    ///
+    /// No-op if `value` is zero, returns success and no events are emitted.
+    ///
+    /// # Errors
+    ///
+    /// Reverts with `InsufficientBalance` if the `value` exceeds the caller's balance.
+    #[ink(message)]
+    fn burn(&mut self, value: u128) -> Result<(), PSP22Error>;
+}
+
+#[ink::trait_definition]
+pub trait PSP22Mintable {
+    /// Mints `value` tokens to the senders account.
+    ///
+    /// # Events
+    ///
+    /// On success a `Transfer` event is emitted with `None` sender.
+    ///
+    /// No-op if `value` is zero, returns success and no events are emitted.
+    ///
+    /// # Errors
+    ///
+    /// Reverts with `Custom (max supply exceeded)` if the total supply increased by
+    /// `value` exceeds maximal value of `u128` type.
+    #[ink(message)]
+    fn mint(&mut self, value: u128) -> Result<(), PSP22Error>;
+}
```

Comment:
I don't think it's written anywhere (certainly not in web3 standards - https://github.com/w3f/PSPs/blob/master/PSPs/psp-22.md) that the selectors are called `PSP22Mintable::mint` and first 4 bytes of hashed ... etc. Since it's not found anywhere, I think we should add it to the README.

Reply:
OK but we should inform about it anyway. Even as little as _"selector will be first 4 bytes of hash taken from `PSP22Burnable::burn` - "0x...""_ is better than nothing

Reply:
+1

Reply:
Done